packages/react-nexus/package.json

```diff
@@ -0,0 +1,46 @@
+{
+  "name": "@bbp/react-nexus",
+  "version": "1.1.0",
+  "description": "A set of React components for Nexus",
+  "keywords": [
+    "react",
+    "nexus",
+    "javascript",
+    "typescript"
+  ],
+  "author": "Julien Machon <julienmachon@gmail.com>",
```

Comment:
You're using your personal email here, should be epfl email (since IP is employer's)

Reply:
oopsy daisy!